src/components/FrontPage/Courses.js

```diff
@@ -8,8 +8,10 @@ import Row from 'react-bootstrap/lib/Row';
 import {getFilteredCourses, getFilteredExternalCourses} from '../../selectors/course';
 import {getTranslator} from '../../selectors/translate';
 import CourseList from '../CourseList/CourseList';
+import {hasPlaylist} from '../../util';
```

Comment:
Hadde det ikke vært bedre om funksjonen faktisk het `coursesWithPlaylists`? For `hasPlaylist` høres litt ut som en boolean...